### PR TITLE
Chore/api reference dbrp

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Database Retention Policy Mappings Endpoints

- Get DBRPs
- Create DBRP
- Get DBRP
- Update DBRP
- Delete DBRP

### CODE REVIEW

File: src/common/paths/dbrps_dbrpID.yml [hunk 1]
```diff
@@ -56,77 +92,135 @@ patch:
...
+
+    - [Database and retention policy mapping]({{% INFLUXDB_DOCS_URL %}}/reference/api/influxdb-1x/dbrp/)
   parameters:
     - $ref: "../parameters/TraceSpan.yml"
     - in: query
       name: orgID
-      description: Specifies the organization ID of the mapping
+      description: |
+        An organization ID.
+        Only returns the specified organization.
```
Comment (jstirnaman) [inline src/common/paths/dbrps_dbrpID.yml:182]:
```suggestion
        Specifies the organization that owns the DBRP mapping.
```

File: src/common/paths/dbrps_dbrpID.yml [hunk 2]
```diff
@@ -56,77 +92,135 @@ patch:
...
       schema:
         type: string
     - in: query
       name: org
-      description: Specifies the organization name of the mapping
+      description: |
+        An organization name.
+        Only returns the specified organization.
```
Comment (jstirnaman) [inline src/common/paths/dbrps_dbrpID.yml:189]:
```suggestion
        Specifies the organization that owns the DBRP mapping.
```

File: src/common/paths/dbrps_dbrpID.yml [hunk 3]
```diff
@@ -56,77 +92,135 @@ patch:
...
       schema:
         type: string
       required: true
-      description: The database retention policy mapping
+      description: |
+        A DBRP mapping ID.
+        Only returns the specified DBRP mapping.
   responses:
     "204":
       description: Delete has been accepted
```
Comment (jstirnaman) [inline src/common/paths/dbrps_dbrpID.yml:202]:
```suggestion
      description: Success. The delete is accepted.
```

Or one of these (not sure which is true):
 - "The DBRP mapping is deleted."
 - "The DBRP mapping is queued for deletion."

File: src/common/schemas/DBRP.yml [hunk 4]
```diff
@@ -2,20 +2,29 @@
   properties:
     id:
       type: string
-      description: The ID of the DBRP mapping.
+      description: |
+        The resource ID that InfluxDB uses to uniquely identify the task.
```
Comment (jstirnaman) [inline src/common/schemas/DBRP.yml:6]:
```suggestion
        The resource ID that InfluxDB uses to uniquely identify the database retention policy (DBRP) mapping.
```

Glossary link?

File: src/common/schemas/DBRPCreate.yml [hunk 5]
```diff
@@ -1,20 +1,30 @@
...
 properties:
-  orgID:
-    type: string
-    description: The ID of the organization.
   org:
     type: string
-    description: The name of the organization that owns this mapping.
+    description: |
+      An organization name.
+      Identifies the [organization]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization) that owns the mapping.  orgID:
```
Comment (jstirnaman) [inline src/common/schemas/DBRPCreate.yml:7]:
```suggestion
      Identifies the [organization]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization) that owns the mapping.
```

File: src/common/schemas/DBRP.yml [hunk 6]
```diff
@@ -2,20 +2,29 @@
...
+        Identifies the InfluxDB v1 database.
     retention_policy:
       type: string
-      description: InfluxDB v1 retention policy
+      description: |
+        A [retention policy](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp) name.
+        Identifies the InfluxDB v1 retention policy mapping.
     default:
       type: boolean
       description: Mapping represents the default retention policy for the database specified.
```
Comment (jstirnaman) [inline src/common/schemas/DBRP.yml:30]:
```suggestion
      description: |
         If set to `true`, this DBRP mapping is the default retention policy
         for the database (specified by the `database` property's value).
```

File: src/common/schemas/DBRPCreate.yml [hunk 7]
```diff
@@ -1,20 +1,30 @@
...
+      Identifies the InfluxDB v1 database.
   retention_policy:
     type: string
-    description: InfluxDB v1 retention policy
+    description: |
+      A [retention policy](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp) name.
+      Identifies the InfluxDB v1 retention policy mapping.
   default:
     type: boolean
     description: Mapping represents the default retention policy for the database specified.
```
Comment (jstirnaman) [inline src/common/schemas/DBRPCreate.yml:30]:
```suggestion
    description: |
        Set to `true` to use this DBRP mapping as the default retention policy
        for the database (specified by the `database` property's value).
```

File: src/common/schemas/DBRPUpdate.yml [hunk 8]
```diff
@@ -1,6 +1,9 @@
   properties:
     retention_policy:
       type: string
-      description: InfluxDB v1 retention policy
+      description: |
+        A [retention policy](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp) name.
+        Identifies the InfluxDB v1 retention policy mapping.
     default:
       type: boolean
+      description: Mapping represents the default retention policy for the database specifi
```
Comment (jstirnaman) [inline src/common/schemas/DBRPUpdate.yml:9]:
```suggestion
      description: |
         Set to `true` to use this DBRP mapping as the default retention policy
         for the database (specified by the `database` property's value).
         To remove the default mapping, set to `false`.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Pull request merged**